### PR TITLE
Prefs [Task][27965]: Handle sort event for Detail Countries Information screen

### PR DESCRIPTION
**Pull request by NguyenTaTrung**
## Related Tickets
- [#Task_27965: Handle sort event for Detail Countries Information screen](https://edu-redmine.sun-asterisk.vn/issues/27965)

## WHAT
- Handle sort event descending, ascending by infected, death and recovered
- Update UI Detail Countries Information screen

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Trainer reivew (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the source code have end-line errors? | | NO |<li>- [x] no</li>|<li>- [ ] no</li>  
Conventions | No hard coding, use constants/configuration values | | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>

### CODE REVIEW

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt [hunk 1]
```diff
@@ -38,6 +42,9 @@ class DetailCountriesFragment :
     override fun initActions() {
         buttonSearch.setOnClickListener { searchCountries() }
         editTextSearch.setOnEditorActionListener(this)
+        textViewInfected.setOnClickListener { sortCountries(TOTAL_CONFIRMED) }
+        textViewDeath.setOnClickListener { sortCountries(TOTAL_DEATHS) }
+        textViewRecovered.setOnClickListener { sortCountries(TOTAL_RECOVERED) }
```
Comment (huytq-2422) [inline app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: Nếu nhiều view set onClick thì mình có thể để Fragment implement View.OnClickListner
rồi xử lý trong hàm onClick()

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt [hunk 2]
```diff
@@ -108,7 +115,101 @@ class DetailCountriesFragment :
...
+
+    private fun updateIcon() {
+        when (stateButton) {
+            STATE_INFECTED_ASC ->
+                textViewInfected.setCompoundDrawablesWithIntrinsicBounds(
+                    R.drawable.ic_baseline_arrow_downward_24,
+                    0,
+                    0,
+                    0
+                )
```
Comment (huytq-2422) [inline app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: sao không viết thêm 1 hàm extention cho gọn =)) 
TextView.setLeftDrawable

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt (hunk 1 above)
Comment (NguyenTaTrung) [reply app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: À ok huy

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt (hunk 2 above)
Comment (NguyenTaTrung) [reply app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: Oki

**Review by aug16vcc — APPROVED** [review]

**Pull request merged**